src/kvstore/raftex/RaftPart.cpp

```diff
@@ -1536,6 +1551,7 @@ void RaftPart::processAppendLogRequest(
         resp.set_committed_log_id(committedLogId_);
         resp.set_last_log_id(lastLogId_);
         resp.set_last_log_term(lastLogTerm_);
+        return;
```

Comment:
Return here is right?  I suggest if we dont' want reset operation, maybe just return is another choice.

Reply:
Yeah, we do need to return in this case.